lib/components/button-close.js

```diff
@@ -41,7 +41,7 @@ export default {
         };
         // Careful not to override the slot with innerHTML
         if (!children.length) {
-            componentData.domProps = { innerHTML: "×" };
+            componentData.domProps = { innerHTML: "&times;" };
```

Comment:
If you look at the history for this file, we were originally inserting `&times;`, which ended up causing the SSR error.

Changing it to the actual UTF-* character `x` works.